mhctools/epitope_collection.py

```diff
@@ -33,7 +33,7 @@ def __init__(
         Collection.__init__(
             self,
             elements=binding_predictions,
-            path=path,
+            sources=[path] if path else [],
```

Comment:
Remind me why you didn't need to do this when switching to `sercol` in https://github.com/hammerlab/varcode/pull/162?